spec/features/landing_face_to_face_search_spec.rb

```diff
@@ -1,35 +1,35 @@
-RSpec.feature 'Consumer searches by postcode only' do
+RSpec.feature 'Landing page, consumer requires general advice in person' do
   let(:landing_page) { LandingPage.new }
   let(:results_page) { ResultsPage.new }
 
-  scenario 'Consumer enters a invalid postcode' do
-    given_i_am_on_the_rad_landing_page
+  scenario 'Using a valid postcode returns firms ordered by distance' do
+    with_elastic_search! do
+      given_i_am_on_the_landing_page
+      and_firms_with_advisers_covering_my_postcode_were_previously_indexed
+      when_i_search_with_a_reading_postcode
+      then_i_am_shown_firms_with_advisers_covering_my_postcode
+      and_i_am_not_shown_non_postcode_searchable_firms
+      and_the_firms_are_ordered_by_distance_in_miles_to_me
+    end
+  end
+
+  scenario 'Using an invalid postcode shows an error message' do
+    given_i_am_on_the_landing_page
     when_i_submit_a_invalid_postcode_search
     then_i_am_told_the_postcode_is_incorrect
   end
 
-  scenario 'Postcode cannot be geocoded' do
+  scenario 'Using a postcode that cannot be geocoded shows an error message' do
```

Comment:
Making all specs consistent, that's all. I wanted to be more explicit in the scenario descriptions and include the expectation of the test as well.